doc/sphinx/source/recipes/recipe_smpi.rst

```diff
@@ -52,11 +53,11 @@ These settings are passed to the other scripts by main.ncl, depending on the sel
 Variables
 ---------
 
-* hfds (ocean, monthly mean, longitude latitude time)
+* hfds (ocean, monthly mean, longitude latitude time) - not available on Jasmin
```

Comment:
Actually the data is available on Jasmin, but not in the CP4CDS selection. Maybe there is no need to mention this here.

Reply:
Changed as suggested.